src/node/cli.ts

```diff
@@ -502,10 +502,21 @@ export async function setDefaults(cliArgs: Args, configArgs?: ConfigArgs): Promi
   } as DefaultedArgs // TODO: Technically no guarantee this is fulfilled.
 }
 
-async function defaultConfigFile(): Promise<string> {
+/**
+ * Helper function to return the default config file.
+ *
+ * @param {string} password - Password passed in (usually from generatePassword())
+ * @returns The default config file:
+ *
+ * - bind-addr: 127.0.0.1:8080
+ * - auth: password
+ * - password: <password>
+ * - cert: false
+ */
+export function defaultConfigFile(password: string): string {
   return `bind-addr: 127.0.0.1:8080
 auth: password
-password: ${await generatePassword()}
+password: ${password}
```

Comment:
🛠️ I believe functions should be _pure_ (unless they can't for some reason) so I refactored this to make `password` a param and pass in `await generatePassword()` when we call this function.

Reply:
Nice!